pkg/workloads/cortex/tf_api/api.py

```diff
@@ -480,13 +498,11 @@ def start(args):
         if api.get("request_handler") is not None:
             local_cache["request_handler"] = ctx.get_request_handler_impl(api["name"])
 
-        if not util.is_resource_ref(api["model"]):
-            if not os.path.isdir(args.model_dir):
-                ctx.storage.download_and_unzip_external(api["model"], args.model_dir)
+        if args.only_download:
```

Comment:
I think the previous logic around this should be preserved, but not entirely sure. It seems like e2e would be broken with this logic (since it still uses the zip, but its init container would not unzip). We can discuss